src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -15,7 +15,8 @@ public class FindCommand extends Command {
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) or phone number that starts with any of the given keywords"
```

Comment:
the word `case-insensitive` might be better written as `not case-sensitive`

Reply:
Updated.